.versionbot/CHANGELOG.yml

```diff
@@ -0,0 +1,542 @@
+- commits:
+    "0":
```

Comment:
This should be an array rather than an object, the rest looks fine though. A good way to test this is to run `balena-versionist` locally on the commit that adds this file. (You should remove the last commit from the file after, as its the one VB will add back) But this way you only need to get it into valid yml, and then balena-versionist will format it properly.

Reply:
Should be OK now @nazrhom